src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -155,7 +155,7 @@ public void onGameTick(GameTick tick) {
         int currentConfigState = config.tilesOffset()
                 + config.warningLimit() * 10
                 * (config.includeTotalLevel() ? -1 : 1)
-                * (config.includeExp() ? -1 : 1);
+                * (config.excludeExp() ? -1 : 1);
```

Comment:
Minor Issue: Technically may miss an update if you were to flip both excludeExp and includeTotalLevel between updates. But unlikely you'd change it that fast (maybe though if you lagged?)
Also tileOffset=10, warningLimit=1 is indistinguishable from tileOffset=0, warningLimit=2.

I don't think we need to handle them in this PR, but I think we could improve this hashing algorithm and move it to the ConfigEval class :+1:

Reply:
Still, great idea. This was a really clever way to do updates

Reply:
Oh yeah definitely this is 100% a quick solution I threw together, It would be much better to just convert all the values into strings. I'll do that now sure.